roles/hitchwiki/tasks/system.yml

```diff
@@ -0,0 +1,64 @@
+---
+# Common tasks
+
+- name: 'install python2' # Ansible needs python or it fails with "/usr/bin/python: not found"
+  become: yes
+  raw: apt-get install -yq python-simplejson
+  # See https://stackoverflow.com/questions/32429259/ansible-fails-with-bin-sh-1-usr-bin-python-not-found
+
+- debug:
+    msg: "root: {{ dir.root }}"
+
+- name: Has configs/settings-example.yml been copied to configs/settings.yml?
+  include_vars: ./configs/settings.yml
+  when: lookup('file', 'configs/settings.yml')
+
+- name: Set language
+  lineinfile: 
+    path: .profile
+    line: |
+      export LC_ALL="en_US.UTF-8"
+      export LANGUAGE="en_US.UTF-8"
+
+- name: Use apt-cacher
+  become: yes
+  lineinfile:
+    path: /etc/apt/apt.conf.d/apt-cacher-ng
+    create: yes
+    regexp: '^Acquire::http::Proxy'
+    line: >
+      Acquire::http::Proxy "http://{{ apt_proxy }}:3142/";
+      Acquire::https::Proxy "false"; # or enable "PassThroughPattern: .*" in /etc/apt-cacher-ng/acng.conf
+      # See https://blog.packagecloud.io/eng/2015/05/05/using-apt-cacher-ng-with-ssl-tls/
+  when: apt_proxy is defined
+
+- name: Remove apt source of NodeSoure 8.x # to avoid dependency problems
```

Comment:
What problems did this cause? Seems like this line is later being put back at Parsoid installation?

Reply:
yes, npm can't be installed when these sources are present.

Reply:
Interesting. Would it work with this:
```
curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
sudo apt-get install -y nodejs
```
[via](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions)

i.e. install nodejs+npm, instead of just npm.

It would be better to have lines referring to node -version all in one place, so that it would be easier to update later on.

Also Maildev and Bower rely on Node.js so it makes sense to install it already here, instead of only at Parsoid step.

Reply:
We are not installing with the bash script, don't really see a different to the repo in your code:
```
- name: Add apt source for NodeSoure 8.x
  become: yes
  apt_repository: filename=nodesource.list state=present repo="deb https://deb.nodesource.com/node_8.x xenial main"

- name: Install nodejs
  become: yes
  apt: name=nodejs state=present update_cache=yes

- name: Link nodejs to node
  become: yes
  file: state=link src=/usr/bin/nodejs path=/usr/local/bin/node

$ sudo apt-get install nodejs npm
Reading package lists... Done
Building dependency tree       
Reading state information... Done
npm is already the newest version (3.5.2-0ubuntu4).
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 nodejs : Conflicts: npm
E: Unable to correct problems, you have held broken packages.
```